modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc

```diff
@@ -0,0 +1,112 @@
+= Creating the default revision tag and relabeling the namespaces
+
+This procedure explains how to create the default revision tag and relabel the namespaces after you have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process using istio injection label.
+
+The `bookinfo` example application is being used for demonstration purposes with a minimal example for the `Istio` resource. For more information on configuration differences between the {SMProduct} 2 `ServiceMeshControlPlane` resource and the {SMProduct} 3 `Istio` resource, see "Configuration fields mapping between Service Mesh 2 and Service Mesh 3".
+
+You can follow these same steps with your own workloads.
+
+.Prerequisites
+
+* You have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process using istio injection label.
+
+.Procedure
+
+. Create a YAML files called `rev-tag.yml` that defines the IstioRevisionTag resource.
++
+.Example `IstioRevisionTag` resource
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: IstioRevisionTag
+metadata:
+  name: default
+spec:
+  targetRef:
+    kind: IstioRevision
+    name: ossm-3-v1-24-3
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f rev-tag.yaml
+----
+
+. Verify that the status of the IstioRevisionTag resource by running the following command:
++
+[source,terminal]
+----
+$ oc get istiorevisiontags
+----
++
+.Example output
+[source,terminal]
+----
+NNAME      STATUS                    IN USE   REVISION        AGE
+default    NotReferencedByAnything   False    ossm-3-v1-24-3  18s
+----
+
+. Add `istio-injection=enabled` label and remove `istio.io/rev` label by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo istio-injection=enabled istio.io/rev-
+----
+
+
+ADD step to repeats steps x to y for each namespace. Delete the step below pertaining to gateways
+
+
+. If you are using gateways, you must migrate them now. For more information, see "Migrating gateways from Service Mesh 2 to Service Mesh 3".
+
+. Restart the workloads by running the following command:
++
+[source,terminal]
+----
+$ oc rollout restart deployments -n bookinfo
+----
+
+. Verify that the `IstioRevisionTag` resource is in use by running the following command:
++
+[source,terminal]
+----
+$ oc get istiorevisiontags
+----
++
+.Example output
+[source,terminal]
+----
+NAME      STATUS    IN USE   REVISION        AGE
+default   Healthy   True     ossm-3-v1-24-3  28s
+----
+
+.Verification
+
+. Ensure that expected workloads are managed by the new control plane by running the following command:
++
+[source,terminal]
+----
+$ istioctl ps -n bookinfo
+----
++
+.Example output
+[source,terminal]
+----
+UPDATE EXAMPLE OUTPUT          
+----
+
+. If the 2.6 installation contains additional namespaces, migrate the next namespace now.
```

Comment:
This should be removed when the "step to repeats steps x to y for each namespace" is added